nautobot/dcim/templates/dcim/device_edit.html

```diff
@@ -24,6 +25,18 @@
         <div class="panel-body">
             {% render_field form.platform %}
             {% render_field form.software_version %}
+            <div class="form-group" id="software_version_image_container" style="display:none">
+                <div class="col-md-3"></div>
+                <div class="col-md-9">
+                    <span class="help-block">
+                        <span id="software_version_image_description"></span>
+                        <ul id="software_version_image_list">
+                            <li> Software images go here </li>
```

Comment:
Is there value in having this `<li>` here or should the list just be empty initially?

Reply:
I just changed this in my local branch. It now looks like this to help describe what it's there for:

```html
            {% render_field form.software_version %}
            <div class="form-group" id="software_version_image_container" style="display:none">
                <div class="col-md-3"></div>
                <div class="col-md-9">
                    <span class="help-block">
                        <span id="software_version_image_description"><img src="{% static 'img/ajax-loader.gif' %}"></span>
                        <ul id="software_version_image_list">
                            <li> Software images for the selected software version will appear here </li>
                        </ul>
                    </span>
                </div>
            </div>
            {% render_field form.software_image_files %}
```

Reply:
BTW, the `<li>` tag is never rendered because of the `style="display:none"` on the parent div and the javascript below.